### DISCUSSION TITLE
Make multiprogress-bar fade out

### DISCUSSION THREAD

**Discussion by aijosh** [cpvalente/ontime#1135] [Q&A] [answered]
Hello,

I'm new to Ontime.

Is there any option available to make the multiprogress-bar fade out.

The current implementation makes the indicator overlay fade from 80% to 0% - left to right.

Since it is a countdown timer, ideally the indicator overlay should fade from 0% to 80% - left to right. This way the progress bar begins to disappear from left to right and the presenter has a visual cue that they are running out of time.

Currently it seems as though the presenter is gaining time (since their bar is getting bolder) instead of loosing time (if their bar was fading out).

Presenter should get a sense of their time fading away as against the current implementation of their time coming back to life.

Is there an option to swap the behaviour of the progress bar overlay?

**Comment by cpvalente** [comment, marked as answer]
Hi @aijosh , thank you for reaching out.

I have been delaying replying to find something meaningful to say. I am unsure if I will succeed.

The current implementation is of a ProgressBar, ie: we show the user how much time has elapsed (progressed) in the allotted time.
This is in contrast of your suggestion of a Countdown bar, which shows how much time is left.

Historically this has been slightly contentious. The first versions of Ontime has a countdown bar as you suggest. Several users were confused and requested that the behaviour was changed. 
At that point we had a view parameter to change the behaviour, this option was later removed in favour of the current progress bar

To answer your question:

> Is there an option to swap the behaviour of the progress bar overlay?
No, the only way forwards is for you to make your own view and create the progress behaviour to match your expectations

This is unlikely to change unless there is significant community support behind the idea.
We try to keep the app simpler and remove logic that is not necessary. Meaning we ideally dont want to maintain multiple mind-sets in ontime.

**Comment by aijosh** [comment]
Thanks for the feedback. Its not a major issue but it'll be good to have the option.

I've tried reading the documentation https://[docs.getontime.no](https://docs.getontime.no/)/ to see the possibility of implementing the said behaviour but the website seems to be unavailable. Not sure if it is really down or my region is blocked.

**Reply by cpvalente** [reply]
The website seems to be fine

https://docs.getontime.no/

Which region are you in? I can check in with the hosting service if there are any region blocks

**Reply by aijosh** [reply]
Africa. Its not opening here. 

I get error
Hmmm… can't reach this page
docs.getontime.no refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED

=================
Just tried using Tor browser now and got it to work with Tor though. But it doesn't open on the normal browser.

**Comment by cpvalente** [comment]
> Thanks for the feedback. Its not a major issue but it'll be good to have the option.
> 
> 
> 
> I've tried reading the documentation https://[docs.getontime.no](https://docs.getontime.no/)/ to see the possibility of implementing the said behaviour but the website seems to be unavailable. Not sure if it is really down or my region is blocked.

Hi! From a quick Google search it seems that some ISPs are blocking vercels address.
See here a ticket for a Nigerian ISP
https://github.com/orgs/vercel/discussions/3630

Moving the hosting could be an option, but not in short term.

Meanwhile I am happy to help with what you may need. If you create a repo I can help
You are also welcome to get in touch at mail@getontime.no
